### ISSUE TITLE
[QUESTION] How do I make an item not used up in a recipe?

### ISSUE THREAD

**Issue by pixe1reality** [stanhebben/MineTweaker#14]
Okay, I'm making a recipe for making metal plates to make things like armour, but how do I make a hammer item not get used up? I'm aware item damaging is in 1.7 (or soon will be) but is it possible to do it in 1.6.4 builds?

**Comment by stanhebben** [comment]
Yes, it's possible in MT3 3.0.2+ with any supported minecraft version. I just haven't updated the wiki yet, but you can look at the 1.7.X tutorials - item names are different but the functions are exactly the same.

**Comment by pixe1reality** [comment]
Cool, thanks!

**Comment by ghost** [comment]
It can be done as so;
recipes.addShaped(<tile.grass>, [[
    <tile.dirt>,
    <item.bucketWater>.transformReplace(<item.bucket>),
    <item.wheat>
]]);
, returning a empty bucket in the crafting table.
